quantum/keycodes.h

```diff
@@ -1365,9 +1393,6 @@ enum qk_keycode_defines {
     SC_LAPO    = QK_SPACE_CADET_LEFT_ALT_PARENTHESIS_OPEN,
     SC_RAPC    = QK_SPACE_CADET_RIGHT_ALT_PARENTHESIS_CLOSE,
     SC_SENT    = QK_SPACE_CADET_RIGHT_SHIFT_ENTER,
-    OU_AUTO    = QK_OUTPUT_AUTO,
```

Comment:
Could we add to `quantum/quantum_keycodes_legacy.h`

```c
#define QK_OUTPUT_AUTO OU_AUTO
```

Reply:
Done.